### PR TITLE
Added photo to workshop blog post

### PR DESCRIPTION
**Pull request by JeffCarver**

### CODE REVIEW

**Comment by JeffCarver** [comment]
It appears to render properly in the preview now.

**Comment by cosden** [comment]
Preview: https://output.circle-artifacts.com/output/job/0e04f90b-3e0c-4f43-b044-b139f587fe14/artifacts/0/intersect-training.github.io/announcement/intersect-RSE-workshop/index.html

(As a side note, it took me a minute to find because it doesn't have the date in the URL - something to remember for next time)

File: _posts/2022-08-19-intersect-RSE-workshop.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ categories:
     - announcement
 tags:
 header: no
+author: carver
+image:
+    title: GroupPhoto.jpg
```
Comment (cosden) [inline _posts/2022-08-19-intersect-RSE-workshop.md]:
```suggestion
    title: Workshop22GroupPhoto.jpg
```

File: _posts/2022-08-19-intersect-RSE-workshop.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ categories:
...
+    thumb: GroupPhoto.jpg
```
Comment (cosden) [inline _posts/2022-08-19-intersect-RSE-workshop.md]:
```suggestion
    thumb: Workshop22GroupPhoto.jpg
```

File: _posts/2022-08-19-intersect-RSE-workshop.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ categories:
...
+    homepage: GroupPhoto.jpg
```
Comment (cosden) [inline _posts/2022-08-19-intersect-RSE-workshop.md]:
```suggestion
    homepage: Workshop22GroupPhoto.jpg
```

**Review by cosden — CHANGES_REQUESTED** [review]
I don't have a good way to suggest the filename change. For the long term, I think we should either have Workshop22 in the filename, or the date (say 2022-08-GroupPhoto.jpg).

**Review by cosden — APPROVED** [review]

**Pull request merged**